appserver/extras/arquillian-containers/payara-embedded/src/test/resources/gfconfigs/instanceRoot/config/readme.txt

```diff
@@ -0,0 +1 @@
+This directory is meant to be the config directory of a Glassfish instanceRoot. It contains a blank domain.xml file for the purpose of testing.
```

Comment:
Weird, I changed it earlier, but changes got lost.

Done: https://github.com/payara/Payara/pull/1658/commits/48e26897b94fd2b4c5b493acb244d72d9d6b3281